secure-api-gateway-ig-extensions/src/main/java/com/forgerock/sapi/gateway/common/jwt/JwtException.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2020-2022 ForgeRock AS (obst@forgerock.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.forgerock.sapi.gateway.common.jwt;
+
+/**
+ * Exception thrown when issues occur processing jwts
+ */
+public class JwtException extends Exception {
+
+    public JwtException() {
+        super();
+    }
+
+    public JwtException(String message) {
+        super(message);
+    }
+
+    public JwtException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    public JwtException(Throwable cause) {
+        super(cause);
+    }
+
+    protected JwtException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace) {
+        super(message, cause, enableSuppression, writableStackTrace);
+    }
```

Comment:
Remove this, if we need an Exception that extends JwtException and needs this functionality then it can add it.